travis/build-and-test-downstream.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -e
+
+build_order=(opm-common opm-parser opm-material opm-core opm-output opm-grid opm-simulators)
```

Comment:
> is the lack of opm-data intentional?

Yes - this is just *building*; however the `opm-data` repository is cloned in the `clone-opm.sh`.